dbms/src/test/java/org/polypheny/db/restapi/RestTest.java

```diff
@@ -181,64 +180,65 @@ private static HttpRequest<?> buildRestDelete( String table, Map<String, String>
     // --------------- Tests ---------------
 
 
-    @Test
-    @Category({ CassandraExcluded.class })
+    @Test // this needs to be rewritten
+    @Tag("hsqldbExcluded")
+    @Tag("monetdbExcluded")
+    @Tag("postgresqlExcluded")
```

Comment:
Why are these excludes necessary?

Reply:
hsqldb and postgres exclude where artifacts, removed.

Monetdb does not correctly handle different timezones and always includes the local one, added a fix for that.